examples/models/customjs.py

```diff
@@ -27,14 +27,14 @@
 plot.add_layout(LinearAxis(), 'below')
 plot.add_layout(LinearAxis(), 'left')
 
-customjs = CustomJS(args=dict(source=source), lang="coffeescript", code="""
+customjs = CustomJS(args=dict(source=source), code=CoffeeScript("""
```

Comment:
This is a breaking change. People are definitely making custom extensions at this point, and @birdsarah and I have both been pushing the idea pretty solidly.  I think this needs a deprecation warning. Sholud be pretty simple, add an  `__init__` to `CustomJS` that strips the `lang` kwarg and converts/wraps `code` accordingly?